### ISSUE TITLE
pandoc-plot-0.9.2.0 release tarball seems to be incomplete

### ISSUE THREAD

**Issue by peti** [LaurentRDC/pandoc-plot#13]
I've tried to build pandoc-plot using the release tarball from Hackage, but my attempt failed during the test suite run apparently because some expected data files are missing:

~~~
  Configuration tests
    empty configuration is correctly parsed to default values:  FAIL
      Exception: LoadSettingsException "tests/fixtures/.empty-config.yml" (InvalidYaml (Just (YamlException "Yaml file not found: /build/pandoc-plot-0.9.2.0/tests/fixtures/.empty-config.yml")))
    example configuration is correctly parsed:                  OK
    Configuration path stored in metadata is correctly parsed:  FAIL
      Exception: LoadSettingsException "tests/fixtures/.config-meta.yml" (InvalidYaml (Just (YamlException "Yaml file not found: /build/pandoc-plot-0.9.2.0/tests/fixtures/.config-meta.yml")))
~~~

Is it possible that those files are missing from the tarball?

**Comment by LaurentRDC** [comment]
Hi Peter!

Yes, I can see that the test files are missing from the Cabal file (and thus the tarball). I'm sorry for that! I will try to correct this tomorrow.

**Comment by LaurentRDC** [comment]
This problem should be fixed for version 0.9.3.0, released today. Let me know how it goes!

**Comment by LaurentRDC** [comment]
I'll close this for now, but feel free to re-open this issue if the problem persists.
